src/main/java/ggs/srr/service/user/FtUserService.java

```diff
@@ -6,16 +6,47 @@
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
+import java.util.*;
```

Comment:
```suggestion
import java.util.*;
```

util에서 필요한 부분들만 임포트 부탁드립니다 :)

Reply:
이부분은 
`import java.util.Collections;
import java.util.HashMap;
import java.util.List;
import java.util.Map;
import java.util.Comparator;
import java.util.ArrayList;`
intellij 에서 일정 수 이상의 util import 시에 *로 초기화시켜서, 다시 수정완료했습니다.

Reply:
이 부분은 평가 포인트 랭킹 시에 수정해서 push해도 괜찮을까요?